src/main/java/backend/github/UpdateIssuesTask.java

```diff
@@ -26,16 +27,21 @@ public UpdateIssuesTask(TaskRunner taskRunner, Repo repo, Model model) {
     public void run() {
         ImmutableTriple<List<TurboIssue>, String, Date> changes = repo.getUpdatedIssues(model.getRepoId(),
             model.getUpdateSignature().issuesETag, model.getUpdateSignature().lastCheckTime);
+        List<PullRequest> updatedPullRequests = repo.getUpdatedPullRequests(
+                model.getRepoId(), "", model.getUpdateSignature().lastCheckTime);
```

Comment:
Are we not using ETags for PRs at all? Given that nothing is passed in here, and PullRequestUpdateService.getUpdatedItems is overridden so it doesn't check them.